ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js

```diff
@@ -182,9 +182,8 @@ export default function SecureYourWallet() {
           {t('seedPhraseIntroSidebarTitleTwo')}
         </Text>
         <Box as="ul" className="secure-your-wallet__list" marginBottom={4}>
+          <Text as="li">{t('seedPhraseIntroSidebarBulletTwo')}</Text>
```

Comment:
`BulletTwo` should come after `BulletOne`, you can reverse the order or rename the variables.

Reply:
Oh I'm confused. It seems the order has been changed back from what I originally proposed. To confirm the bullets should read as follows in the following order:
- Write down and store in multiple secret places
- Store in a deposit box

Reversing the order in the HTML seemed to be the quickest way, which is what I originally did, than changing the variable names in all of the different language files.

Reply:
Resolved in [8e27091](https://github.com/MetaMask/metamask-extension/pull/25985/commits/8e2709110332609d8bc6bb2dd2d89f1f658b57b5)